### PR TITLE
Homework Assignment 01 from MBrower

### PR DESCRIPTION
**Pull request by mtbrower**

### CODE REVIEW

**Comment by gummywormz** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       X       |            |
| **Coding strategy**         |           |       X       |            |
| **Presentation**            |     X      |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |           |    X          |            |

Remarks:

Good use of markdown elements.
Make your commit titles and descriptions more descriptive and substantive. This makes it easy for other people and yourself to easily know what changes you made in case there is a conflict or error that needs to be resolved or rolled back.

**Comment by rachelyhkim** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |     x         |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |           |      x        |            |
| **Achievement, creativity** |           |      x        |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* I love the picture of your cat! I also like how often you committed your file—I think that's a really good habit to build.
* I would try to be more intentional in when/where you are using style decisions such as italics. For example, the purpose of having "I also have a cat named Milo" in italics is a bit unclear. I also feel similarly about the "< About me > tag" header. The use of the brackets made me read it as if it was a line of code that accidentally got transferred into the final document.
* Finally, I would also try to do a brief proofread to catch errors in spelling or capitalization.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
+
+
 # hw01
 Repository for homework 01
+
+To prepare this following homework assignment, I first needed to create a readme file that was cloned in order to link R Studio to my repository. After formating the below text using the appropriate syntax, I double checked that I forked the repository for homework 1, I committed the text I wrote in R script, and selected the push button to add this text and image to my repository via R. 
+
+## <**About me**> tag
```
Comment (mayank-kgp) [inline README.md:8]: If it was not intentional then just 
### About me
would have been better.

**Comment by mayank-kgp** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |           |     x         |            |
| **Ease of access**          |       x    |              |            |

Remarks:

* You were supposed to write about how you changed the readme.md in the [description box](https://github.com/uc-cfss/hw01/pull/22#issue-220976389). 
* Commit messages could have been more logical rather than "k" or "these are new changes". However, this factor is not counted in the evaluation.

**Comment by mayank-kgp** [comment]
One more comment - You don't need to commit all files such as .gitignore or .Rproj files. Just commit those files which are necessary. In this case, only 2 files are required - .md file and image file.

**Pull request closed without merging**